### DISCUSSION TITLE
Dịch "mean squared"

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#83] [Terminology] [answered]
"mean squared" mình thấy đang không nhất quán giữa 2 cách dịch
👍  **trung bình bình phương**
và
😄 **toàn phương trung bình** 

Một số context: 
- mean squared error
- mean squared distance

Anh @ngcthuong đề xuất luôn dịch là "toàn phương trung bình". Mn thấy sao?
cc @mlbvn/handson-ml2

**Comment by ngcthuong** [comment]
Từ mean squared mình thấy dùng phổ biến rồi. Mình cứ dùng cách dịch thông thường thôi.

**Reply by forummlcb** [reply]
Mình vote `trung bình bình phương` nhé. Theo ý nghĩa toán học thì nó là trung bình của các bình phương (lỗi). 

MSE: Trung bình bình phương lỗi.
RMSE: Căn bậc hai trung bình bình phương lỗi.

Theo mình, từ `toàn phương` là `quadratic`, không đúng trong trường hợp này.

**Comment by minhduc0711** [comment, marked as answer]
https://github.com/mlbvn/handson-ml2-vn/discussions/83#discussioncomment-677402
> Mình vote trung bình bình phương nhé. Theo ý nghĩa toán học thì nó là trung bình của các bình phương (lỗi).
>
> MSE: Trung bình bình phương lỗi.
> RMSE: Căn bậc hai trung bình bình phương lỗi.
>
>Theo mình, từ toàn phương là quadratic, không đúng trong trường hợp này.
